client/my-sites/email/email-management/home/mailpoet-upsell.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, CompactCard } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import mailpoetSymbol from 'calypso/assets/images/email-providers/mailpoet-symbol.svg';
+import QueryJetpackPlugins from 'calypso/components/data/query-jetpack-plugins';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getPluginOnSite, isRequesting } from 'calypso/state/plugins/installed/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+export default function MailPoetUpsell() {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+	const isFetchingPlugins = useSelector( ( state ) => isRequesting( state, selectedSite?.ID ) );
+	const mailpoetPlugin = useSelector( ( state ) =>
+		getPluginOnSite( state, selectedSite?.ID, 'mailpoet' )
+	);
+
+	return (
+		<>
+			<QueryJetpackPlugins siteIds={ [ selectedSite?.ID ?? 0 ] } />
+
+			{ ! isFetchingPlugins && ! mailpoetPlugin && (
+				<div className="mailpoet-upsell">
+					<CompactCard className="mailpoet-upsell__card">
+						<div className="mailpoet-upsell__icon">
+							<img src={ mailpoetSymbol } alt="MailPoet logo" />
+						</div>
+
+						<div className="mailpoet-upsell__text">
+							<h4>MailPoet</h4>
+							<span>
+								{ translate(
+									'Create and send beautiful newsletters, post notifications, welcome emails and more from WordPress.com.'
```

Comment:
We discussed some different alternatives for this text blurb previously. I'm open to changing it if @semihakocer or @wensco have suggestions, but I think we should aim for something that fits on a single row on a 13" laptop monitor.

Reply:
@fredrikekelund what's the character limit to fit in a single line? We can modify the text accordingly.

Reply:
It's not an exact character limit since different characters can render at different widths, but it's somewhere around **120 characters**.

If you use the [Calypso live link](https://calypso.live/?image=registry.a8c.com/calypso/app:build-39420) and then follow the Testing instructions in the PR description, you can verify that a particular string looks good on laptop, if you like (by changing the text through the browser dev tools).

Reply:
Some alternatives for your to pick:

Create and send beautiful newsletters, post notifications, and more with MailPoet.

Send beautiful newsletters, post notifications, emails, and more with MailPoet.

Design and send newsletters to your followers with MailPoet.

Reply:
Looks good